_posts/2020-11-07-What-is-Tmux.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "What is Tmux?"
+date:   2020-11-07 12:54:04 -0700
+categories: Tmux Think Create Features
+---
+![tmux Internet Image](https://github.com/tmux/tmux/raw/master/logo/tmux-logo-medium.png?raw=true)
+
+Tmux is an essential program for working with remote servers over SSH. SSH will drop the connection if a person's network fails or if they don't interact with the window for an extended period of time. This is significant because anything the person was running will stop, regardless if it is done or not. So, in order to avoid this mess, a person must "hand off" the work to tmux. The program will allow the process to continue even after they disconnect, whether it be on purpose or on accident.
+
+**Tmux interface: on the command line, plain text**
+
+
+## Think About It This Way
+Let's say a student and their team must complete a presentation in two days, but this student has to leave and cannot access the slides after the first day. In order to complete the task on time, they must use a program such as Google Slides. This will allow the student to hand off the presentation to another person who can complete the task while they are away.
+
+*Tmux ≈ Google Slides + the team who will complete the task*
+
+
+## How To Create And Connect To Tmux
+
+1) Start a new session
+*Note: You can have as many tmux sessions running at a time! The work is saved and stays the same when you leave a particular session*
+
+![Create tmux session screenshot]({{ site.url }}/assets/tmux-01.jpg)
+
```

Comment:
It might be helpful to explain more explicitly what the code will do. So you could say something like: this code will create a new tmux sesssion and attach to it called introduction-tmux

Reply:
Okay, thanks!